src/feedback/vue/submitFeedback.vue

```diff
@@ -82,9 +86,14 @@ export default defineComponent({
             this.isSubmitting = true
             console.log('Submitting feedback...')
             // identifier to help us (internally) know that feedback came from either CodeWhisperer or AWS Toolkit
+            let userComment =
+                this.feedbackName === 'CodeWhisperer' ? 'CodeWhisperer onboarding: ' + this.comment : this.comment
+            // info to help us (internally) know the jobId and clientId for transform by Q job runs
+            this.extraCodeTransformData['jobId'] = transformByQState.getJobId()
```

Comment:
`jobId` could mean anything. This feedback is going into a place where all Toolkit feature goes. 

Can we check `feedbackName == 'AmazonQTransform'` or whatever, so that the `transformByQState` data is not included randomly in all feedback.

Reply:
Makes sense. I wasn't sure where the feedbackName is set though?

Reply:
Will ask PR author who implemented the feedbackName functionality

Reply:
Spoke to @laileni and actually https://github.com/aws/aws-toolkit-vscode/blob/f8606ccab4b209aae0cb5d6aa2bc49e8bbf726ae/src/feedback/vue/submitFeedback.vue#L87 is redundant / doesn’t actually do anything. They removed the “submit feedback” button from their welcome / info page so this no longer does anything

It looks like all feedback submitted through IDE goes through the generic AWS Toolkit page. What I can do is only include the jobId if it is not empty i.e. the user must have clicked Transform.

Reply:
As the CW Onboarding page has low engagement with users so, @ankit-sharma90 suggested to remove the content from the page. I think @Will-ShaoHua  did the clean up work for VSC pre re invent. 

Previously, we used have a link "`Provide Feedback`" which uses common feedback component but I used above logic to separate CW f/b with toolkit f/b.
As per my understanding this logic is redundant right now after new CW Getting started page.

Reply:
> this logic is redundant right now after new CW Getting started page.

The idea is that some features have their own "Send Feedback" action which sends this `feedbackName`. That could be implemented for Transform, or more generally for Q. So when "Send Feedback" is selected from the Q panel and other Q contexts, it sends `feedbackName=AmazonQ`, then we can decide extra logic here.

For reference, look for `feedbackName` in https://github.com/aws/aws-toolkit-vscode/commit/398744e7d465f660067cf073b5ba94dc466fb9a3 . You would need to implement something that like for Transform.

> What I can do is only include the jobId if it is not empty i.e. the user must have clicked Transform.

That's an ok compromise for now.